tools/kagami/src/main.rs

```diff
@@ -57,6 +57,8 @@ pub enum Args {
     Docs(Box<docs::Args>),
     /// Generate a list of predefined permission tokens and their parameters
     Tokens(tokens::Args),
+    /// Generate an example instruction based on the chosen usecase
```

Comment:
```suggestion
    /// Generate an example instruction based on the chosen use case
```

Reply:
Would appreciate the same wording in `README.md`, should help with clarity.